content/applications/marketing/events/event_booths.rst

```diff
@@ -0,0 +1,271 @@
+
+When a person purchases a booth rental through the event website, the subsequent renter-related
+fields on the form auto-populate, based on the information provided by the purchaser during the
+online transaction. The booth also automatically changes its status from *Available* to
+*Unavailable*.
+
+However, if the rental of a booth is conducted in any other way (e.g., in person, via sales order,
+etc.), the :guilabel:`Renter`, :guilabel:`Renter Name`, :guilabel:`Renter Email`, and
+:guilabel:`Renter Phone` fields can be entered in manually.
+
+The status of the booth (:guilabel:`Available` or :guilabel:`Unavailable`) can also be changed
+manually, either by clicking the appropriate status from the status bar present on the booth form,
+or by dragging-and-dropping the desired booth into the appropriate stage, via the *Booths* page
+Kanban view.
+
+Sell event booths
+=================
+
+With event booths configured in the back-end, via the event-specific *Booths* pages, Odoo presents
+them on the front-end of the event website, via the *Get A Booth* event subheader link.
+
+To access the *Get A Booth* page on the event website, open the :menuselection:`Events app`, and
+select the desired event from the :guilabel:`Events` dashboard. From the event form, click the
+:guilabel:`Go to Website` smart button to be taken to the front-end of the Odoo-built event website.
+
+From there, click the :guilabel:`Get A Booth` event subheader menu option. Doing so reveals the
+:guilabel:`Get A Booth` page, showcasing all the configured event booths that were created in the
+back-end.
+
+.. image:: event_booths/get-a-booth-page.png
+   :align: center
+   :alt: Typical Get A Booth page on the front-end of the event website via the Odoo Events app.
+
+.. note::
+   If the event subheader menu (with the :guilabel:`Get A Booth` option) is *not* showing up on the
+   event website, there are two ways to make it appear.
+
+   #1. While on the front-end of the event website, enter the edit mode by clicking the
+   :guilabel:`Edit` button in the upper-right corner. Then, click into the :guilabel:`Customize` tab
+   of the resulting sidebar of web design tools.
+
+   In the :guilabel:`Customize` tab, click the toggle switch for :guilabel:`Sub-Menu (Specific)`,
+   and click :guilabel:`Save`. Doing so reveals the event subheader menu with various options.
+
+   #2. Enter :doc:`Debug mode <../../general/developer_mode>`, and open the specific event form in
+   the back-end of the *Events* application.
+
+   On the event form, with *Debug mode* on, an array of subheader menu options appears. Tick the
+   checkbox for :guilabel:`Website Submenu`, in order for the submenu to appear on the event
+   website. Doing so also ticks every other submenu-related checkbox automatically.
+
+   At this point, proceed to choose which options to keep on the event subheader menu. In this case,
+   make sure the :guilabel:`Booth Register` checkbox is ticked.
```

Comment:
This content looks great, and I'm glad you were able to figure out what makes the event subheader appear :smile: 

Given that we're using a pre-configured runbot, the Event subheader menu appears by default. However, for general users creating a new, blank event, this subheader menu won't be visible. To align with their experience, I suggest moving this information out of the note admonition and placing it below your instruction to click the "Go to Website" button. After enabling the event subheader, we can guide them to click the "Get a Booth" button.